Platform.PCL/Realm.PCL/Schema/ObjectSchemaPCL.cs

```diff
@@ -23,18 +23,20 @@
 namespace Realms.Schema
 {
     /// <summary>
-    /// Public description of a class stored in a Realm, as a collection of managed Property objects.
+    /// Public description of a class stored in a <see cref="Realm"/>, as a collection of managed <see cref="Property"/> objects.
     /// </summary>
     public class ObjectSchema : IReadOnlyCollection<Property>
     {
         /// <summary>
         /// Gets the name of the original class declaration from which the schema was built.
         /// </summary>
+        /// <value>The name of the class.</value>
```

Comment:
Are these necessary? Does lacking them make the docs look incomplete or something? They seem really redundant to me.